### PR TITLE
SUPESC-315 Fix CMS Block connector modules integration guides

### PR DESCRIPTION
**Pull request by AsonUnique**
#### Overview:

- Jira ticket: https://spryker.atlassian.net/browse/SUPESC-315

### CODE REVIEW

**Review by DmitryLymarenko — DISMISSED** [review]

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md [hunk 1]
```diff
@@ -17,295 +17,184 @@ related:
...
+    protected function getCmsBlockViewPlugins(): array
+    {
+        return array_merge(parent::getCmsBlockViewPlugins(), [
+            new CmsBlockCategoryListViewPlugin(),
+        ]);
+    }
 }
 ```
 
 **To configure the block:**
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:173]: Do I understand correctly that this is the procedure of adding a CMS block to the category pages?

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md [hunk 2]
```diff
@@ -17,295 +17,184 @@ related:
...
-        ]);
+        return [
+            new CmsBlockCategoryRelationReadPlugin(),
+        ];
     }
 }
 ```
 
-13. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
+6. Optional: To show which categories are assigned to a block on a block view page, register the category list plugin by adding `CmsBlockCategoryListViewPlugin` to the CMS Block GUI dependency provider:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:149]: Is that in the Back Office > Content > Blocks > View ?

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md [hunk 3]
```diff
@@ -17,295 +17,184 @@ related:
...
-4. Set the "Category" and enter the category URL in the Category field. While typing, the product  search will offer suggestions from the product list.
-5. View on a CMS Block edit page.
-6. View on a Category edit page.
-7. Set the block to active to use it straight away.
-8. After clicking **Save**, you will be prompted to provide glossary keys for the placeholders  included in the Twig template.
-9. Embed the block into the category page by adding the following code in the `catalog.twig` template:
+1. In the Backoffice, go to the Content section and navigate to the Blocks section.
+2. Click Create Block to create a new block.
+3. From the template drop-down, select the "Title and description block" template and name the new block.
+4. Select the Categories in the "Categories: Top", "Categories: Middle" and "Categories: Bottom" fields. While typing, the product search will offer suggestions from the product list.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:178]: Does it offer suggestions from the product or from the category list? As I understand, we are assigning a cms block to categories

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md [hunk 4]
```diff
@@ -17,295 +17,184 @@ related:
...
-3. From the template drop-down, select the new template and  name the new block.
-4. Set the "Category" and enter the category URL in the Category field. While typing, the product  search will offer suggestions from the product list.
-5. View on a CMS Block edit page.
-6. View on a Category edit page.
-7. Set the block to active to use it straight away.
-8. After clicking **Save**, you will be prompted to provide glossary keys for the placeholders  included in the Twig template.
-9. Embed the block into the category page by adding the following code in the `catalog.twig` template:
+1. In the Backoffice, go to the Content section and navigate to the Blocks section.
+2. Click Create Block to create a new block.
+3. From the template drop-down, select the "Title and description block" template and name the new block.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:177]: Shouldn't it be named something like "category block"?

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md [hunk 5]
```diff
@@ -17,295 +17,184 @@ related:
...
 {% raw %}{%{% endraw %} endif {% raw %}%}{% endraw %}
 ```
 
-10. To see the page in Yves, the client data storage (Redis) must be up-to-date. This is handled through `cronjobs`.
-11. To manually execute this step,  run the collectors to update the frontend data storage:
-
-```bash
-vendor/bin/console collector:storage:export
-```
+8. To see the page in Yves, the client data storage (Redis) must be up-to-date. This is handled through `cronjobs`.
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:189]: Please provide the cron job to run.

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md [hunk 6]
```diff
@@ -17,295 +17,184 @@ related:
...
 
 CMS Block positioning means that you can predefine some of the useful places in your Twig templates once and then manage your CMS Blocks based on relations to categories and position. For example, you could define "header", "body", "footer" positions to manage your CMS Blocks in those places independently.
 
-By default we provide the following positions: "Top", "Middle", "Bottom", but you can easily change them in the module configuration on a project level (put your extension of `CmsBlockCategoryConnectorConfig` with the replaced method `getCmsBlockCategoryPositionList` to `Pyz\Zed\CmsBlockCategoryConnector\CmsBlockCategoryConnectorConfig` as in the example below).
+By default, we provide the following positions: "Top", "Middle", "Bottom", but you can easily change them in the module configuration on a project level (put your extension of `CmsBlockCategoryConnectorConfig` with the replaced method `getCmsBlockCategoryPositionList` to `Pyz\Zed\CmsBlockCategoryConnector\CmsBlockCategoryConnectorConfig` as in the example below).
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:197]: Please divide this into several sentences. The second sentence should be a direct step to achieve the goal

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-product-cms-blocks.md [hunk 7]
```diff
@@ -24,122 +24,118 @@ related:
...
+            new CmsBlockProductAbstractListViewPlugin(),
+        ]);
     }
 }
+
 ```
 
-6. Register the block list to the product abstract view (optional).
-To show which blocks are assigned to a product abstract on a product abstract view page, add `CmsBlockProductAbstractBlockListViewPlugin` to the Product Management dependency provider.
+5. Optional: To show which blocks are assigned to a product on a product abstract view page, register the block list to the product abstract view by adding `CmsBlockProductAbstractBlockListViewPlugin` to the Product Management dependency provider:
```
Comment (andriitserkovnyi) [inline docs/scos/dev/technical-enhancement-integration-guides/integrating-product-cms-blocks.md:116]: `register the block list to the product abstract view` seems incorrect. Anything we can improve here?

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md (hunk 1 above)
Comment (AsonUnique) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:173]: Yes

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md (hunk 2 above)
Comment (AsonUnique) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:149]: Yes

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-product-cms-blocks.md (hunk 7 above)
Comment (AsonUnique) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-product-cms-blocks.md:116]: I think we can simply remove this line

File: docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md (hunk 4 above)
Comment (AsonUnique) [reply docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md:179]: We don't have such template: 
Screenshot 2022-06-09 at 09 59 28

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**